plugins/database/mysql/connection_producer.go

```diff
@@ -37,8 +37,6 @@ type mySQLConnectionProducer struct {
 	Password                 string      `json:"password" mapstructure:"password" structs:"password"`
 	AuthType                 string      `json:"auth_type" mapstructure:"auth_type" structs:"auth_type"`
 	ServiceAccountJSON       string      `json:"service_account_json" mapstructure:"service_account_json" structs:"service_account_json"`
-	usePrivateIP             bool        `json:"use_private_ip" mapstructure:"use_private_ip" structs:"use_private_ip"`
-	usePSC                   bool        `json:"use_psc" mapstructure:"use_psc" structs:"use_psc"`
```

Comment:
Do we still want these fields for mysql?

Reply:
Yes, I havee tested them as well.